src/uniprotkb/components/protein-data-views/DiseaseInvolvementView.tsx

```diff
@@ -15,6 +15,21 @@ import { Namespace } from '../../../shared/types/namespaces';
 import { FeatureDatum } from './UniProtKBFeaturesView';
 
 import styles from './styles/variation-view.module.scss';
+import externalUrls from '../../../shared/config/externalUrls';
+
+export const uniprotVariantLink = (variant: FeatureDatum) =>
+  variant.alternativeSequence?.originalSequence ||
+  variant.alternativeSequence?.alternativeSequences?.[0] ? (
+    <ExternalLink url={externalUrls.UniProt(variant.featureId || '')} noIcon>
+      {variant.alternativeSequence?.originalSequence || <em>missing</em>}
+      {'>'}
+      {variant.alternativeSequence?.alternativeSequences?.[0] || (
+        <em>missing</em>
+      )}
+    </ExternalLink>
+  ) : (
+    <em>missing</em>
+  );
```

Comment:
I think all of them should be wrapped with the external link component, even if they are "missing". In the page example you gave, the first displayed variant had "missing" written but still had a featureId in the data, so it should still link the the corresponding page.

Reply:
But it takes to the expasy home page. I assume that doesn't exist. 
this has a change - -https://web.expasy.org/variant_pages/VAR_084833.html
while this has 'missing' - https://web.expasy.org/variant_pages/VAR_012494.html

Reply:
I checked how it is done in legacy. It is not linked there either

Reply:
Ah, right. I just saw that the ID is there in the data, but didn't realise the corresponding link in the expasy site was leading nowhere.
In which case everything is fine 👌